### PR TITLE
DAY 2 Add EC2 instance

### PR DESCRIPTION
**Pull request by Coding-Pinata**
DAY 2 Follow along 1 - write the first ec2 instance definition

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.terraform.lock.hcl
+.terraform/
+terraform.tfstate
+terraform.tfstate.backup
```
Comment (geoffrey1330) [inline .gitignore]: could you  please, add newline at the end of file

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+provider "aws" {
+    region = "us-east-1"
+}
+
+resource "aws_instance" "app_server" {
+    ami           = "ami-0715c1897453cabd1"  # Amazon Linux 2023 AMI, free tier eligible official Amazon linux image.
+    instance_type = "t3.micro" 
+}
```
Comment (geoffrey1330) [inline main.tf:18]: could you  please, add newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (RustyLobster) [reply .gitignore]: Added a new line EOF.

File: main.tf (hunk 2 above)
Comment (mariachiinajar) [reply main.tf:18]: Added a new line EOF.

Comment (mariachiinajar) [reply main.tf:18]: @geoffrey1330 
This `main.tf` file will be deleted. 
`.gitignore` is updated with the new line. 
Can you please approve this PR?

**Pull request merged**